src/components/AddApiKey.tsx

```diff
@@ -102,8 +64,12 @@ export default class AddApiKeys extends Component<AddApiKeysProps, AddApiKeysSta
 						name: '',
 						type: 0,
 						working: false,
-						recentlyAddedApiKeys: [...state.recentlyAddedApiKeys, newKey],
 					}))
+					if (!window.recentlyAddedApiKeys){
+						window.recentlyAddedApiKeys = []
+					}
+					window.recentlyAddedApiKeys[newKey.id] = newKey.token
```

Comment:
It is not a good practice to use `Array` with id as index - You should use `object` for that.

Reply:
done